docs/remote/devcontainer-cli.md

```diff
@@ -126,6 +159,97 @@ For example, you may want to pre-build several images that you then reuse across
 
 That's it!
 
+### Building Multi-Architecture builds
+
+#### Pre-requisites
+1. [Node.js 14+](https://nodejs.org).
+2. [The `docker` CLI](/docs/remote/containers#installation).
+3. [Docker `Buildx`](https://docs.docker.com/buildx/working-with-buildx/)
+4. Docker logged in to a Dockerhub registry.
```

Comment:
I would say just add one more sentence explaining why this is (aka. because the multi-platform build option pushes to a container registry by default so the containers are accessible).